packages/twenty-front/src/modules/object-record/record-field/types/guards/assertFieldMetadata.ts

```diff
@@ -51,19 +51,17 @@ type AssertFieldMetadataFunction = <
                           ? FieldNumberMetadata
                           : E extends 'PHONE'
                             ? FieldPhoneMetadata
-                            : E extends 'PROBABILITY'
```

Comment:
let's rename 'PROBABILITY' to 'RATING' here but keep it

Reply:
there's already a 'RATING' just a little above